packages/odyssey-lint-staged/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@okta/odyssey-lint-staged",
+  "version": "0.0.0",
+  "main": "src/index.js",
+  "license": "Apache-2.0",
+  "dependencies": {
```

Comment:
don't forget `"author"` and `"private"` for both of your new packages here!

Reply:
Good catch, has been added to both files.